src/toil/batchSystems/lsf.py

```diff
@@ -297,6 +298,35 @@ def parseBjobs(self,bjobs_output_str):
 
             return bjobs_records
 
+        def parseMaxMem(self, jobID):
+            """
+            Parse the maximum memory from job.
+
+            params:
+            jobID: ID number of the job
+            """
+            memargs = ["bjobs", "-l", str(jobID)]
+            try:
+                bjobs = subprocess.check_output(memargs, universal_newlines=True)
+                memregex = r"MAX MEM: (.*?);"
+                meminfo = re.search(memregex, bjobs)
+                s = " ".join(bjobs.split())
+                command = re.search(r"Command <(.*?)>", s)
+                if meminfo:
+                    if not command:
+                        logger.info("Missing Command: %s", bjobs)
```

Comment:
Consider a more verbose: `Cannot Parse Max Memory Due to Missing Command String`.